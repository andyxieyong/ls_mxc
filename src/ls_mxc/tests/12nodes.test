#NbNodes
12

#NbCores
2

#Deadline
20

#C_LO
1
7
2
3
1
1
3
2
2
1
3
1

#C_HI
2
0
4
6
0
0
6
0
4
2
6
0

#Pred
#A  B  C  D  E  F  G  H  I  J  K  L
0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 
1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 
1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0
0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0
0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0
0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0
0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0        
0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0        
1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 0