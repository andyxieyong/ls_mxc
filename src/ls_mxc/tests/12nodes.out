============= S HI =================

|A | A | D | D | D | D | D | D | G | G | G | G | G | G | I | I | I | I | 0 | 0 | 
|0 | 0 | C | C | C | C | K | K | K | K | K | K | 0 | 0 | J | J | 0 | 0 | 0 | 0 | 


============= S LO =================

|A | B | C | C | B | B | K | K | K | B | B | F | H | H | L | 0 | 0 | 0 | 0 | 0 | 
|0 | D | D | D | G | G | B | B | G | I | I | E | J | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 
